src/wwwroot/template-editor/adapterService.js

```diff
@@ -23,7 +23,8 @@
       getPreviewImageWithPlay: getPreviewImageWithPlay,
       getSettings: getSettings,
       refreshMercadoShopsProductServiceToken: refreshMercadoShopsProductServiceToken,
-      getTiendaNubeProducts: getTiendaNubeProducts
+      getTiendaNubeProducts: getTiendaNubeProducts,
+      overrideUploadFiles: overrideUploadFiles
```

Comment:
Could you help to me telling me where is being called the adapterService method overrideUploadFiles? I check (https://github.com/MakingSense/MSEditor/pull/1259) and I can't found it. I can see that dropzone.uploadFiles is being overrided but I can't found where dropzone.uploadFiles is called.

Reply:
Oh, it is called by dropzone, let me see...

Here it is: https://github.com/enyo/dropzone/blob/caf200c13fd3608dd6bed122926d5848927f55b4/dist/dropzone.js#L2272